tests/foreman/ui/test_contenthost.py

```diff
@@ -4,9 +4,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: Hosts-Content
+:CaseComponent: ContentManagement
```

Comment:
```suggestion
:CaseComponent: HostsContent
```
` HostsContent` looks more related because in bugzilla we have component name 'Hosts-Content' , please check

Reply:
Yes, we are requested to "Remove all characters outside of [-_a-zA-Z] range."

Reply:
I see "Test class for Content Hosts UI" in that file. So, leave it as Hosts-Content. We are we are requested to "Remove all characters outside of [-_a-zA-Z] range.", so the hyphen should stay in "Hosts-Content".